DESCRIPTION

```diff
@@ -1,48 +1,51 @@
-Package: PACTA.analysis
-Title: Helpers for 'PACTA_analysis'
+Package: PACTA.analysis.dependencies
+Title: Install Depdendencies of PACTA_analysis
 Version: 0.0.0.9000
 Authors@R: 
-    c(person(given = "Mauro",
-             family = "Lepore",
-             role = c("aut", "ctr", "cre"),
-             email = "maurolepore@gmail.com",
-             comment = c(ORCID = "0000-0002-1986-7988")),
-      person(given = "Clare",
-             family = "Murray",
-             role = "aut",
-             email = "clare@2degrees-investing.org"),
-      person(given = "Jacob",
-             family = "Kastl",
-             role = "aut",
-             email = "jacob.kastl@gmail.com"),
       person(given = "2 Degrees Investing Initiative",
-             role = c("cph", "fnd"),
+             role = c("aut", "ctr", "cre"),
              email = "contact@2degrees-investing.org"))
```

Comment:
I think there's a mismatched bracket here actually

Reply:
Style: 
* for functions with arguments spanning multiple lines, open paren at end of line, close paren on own line.
* For functions with more than 2 arguments, we should split to multiple liines, with 1 argument per line.
* Name your arguments

examples:
```r
# good
foo <- bar(x = bax, y = "bay", z = c(baz, baw))

# better
foo <- bar(
  x = bax,
  y = "bay",
  z = c(baz, baw)
)

# don't do this
foo <- bar(bax,
  "bay",
  z = c(baz, baw))
```